src/components/index.ts

```diff
@@ -1,12 +1,12 @@
-export { default as AsyncTable } from './AsyncTable';
-export { default as Button } from './Button';
-export { default as Card } from './Card';
-export { default as DropdownMenu } from './DropdownMenu';
-export { default as Input } from './Input';
-export { default as LoadingIndicator } from './LoadingIndicator';
-export { default as Modal } from './Modal';
-export { default as Skeleton } from './Skeleton';
+export { AsyncTable } from './AsyncTable';
+export { Button } from './Button';
+export { Card } from './Card';
+export { DropdownMenu } from './DropdownMenu';
+export { Input } from './Input';
+export { LoadingIndicator } from './LoadingIndicator';
+export { Modal } from './Modal';
```

Comment:
We exported only `Modal` here. I am pretty sure we can use `export * from './Modal';` here as well. Then we can use `import { Modal } from '@zero-tech/zUI/components';`  and `import type { ModalProps } from '@zero-tech/zUI/components';`

Reply:
Yeah, Exactly! We could use `ModalProps` in zApps components for type definition or some type references.

Reply:
Yeah good point, we could use `export * from './Modal'` here. Will wait on @jtulloch to review before pulling the trigger.

Reply:
yeah. i think it's worth being a *bit* careful what gets propagated into the package interface as a whole, and would lean more towards what @colbr said in the earlier comment about making exports explicit. but at the same time. .. probably not a huge deal, really, as i doubt we'll be exporting much at the component level that we wouldn't want at the package level.

Reply:
Awesome thanks for the input guys - I'll use `export *` for these and merge. Cheers!